C/frame.c

```diff
@@ -1,13 +1,13 @@
 #include "frame.h"
 
-#define READ_(bits)                                                                                                           \
+#define READ_(bits,size)                                                    \
```

Comment:
1f245eccc7593859bcdcbfe24d0c09f4c8443867: We should add an explanation to the docs that `size` is the bit width of the target data type which is returned. `bits ≤ size` must hold. We could add a static assert.